config/locales/vi.yml

```diff
@@ -0,0 +1,36 @@
+vi:
+    static_pages:
+      app_name: "Sample App"
+      welcome: "Chào mừng đến với Sample App"
+      signup_now: "Sign up now"
+      home: "Home"
+      help: "Help"
+      about: "About"
+      signup: "Sign Up"
+      signupnow: "Sign up now!"
+      contact: "Contact"
+      login: "Log in"
+      users: "Users"
+      profile: "Profile"
+      settings: "Cài đặt"
+      logout: "Log out"
+      this_is_home_page: "This is the home page for the"
+      same_app: "sample application."
+      contact_the_ruby: "Contact the Ruby on Rails Tutorial about the sample app at the"
+      contact_page: "contact page"
+      railstutorial: "Ruby on Rails Tutorial"
+      base_title: "Ruby on Rails Tutorial Sample App"
+      rails_news: "http://news.railstutorial.org/"
+      rails_link: "http://www.railstutorial.org/"
+      rails_scr: "http://screencasts.railstutorial.org/"
+    users:
+      new:
+        create: "Tạo tài khoản"
+        confirmation: "Xác nhận"
+        user_not_found: "User not found"
+      shared:
+        form_contains: "The form contains"
+        error: "error"
+      sessions:
+        invalid: "Invalid email/password combination"
```

Comment:
Why in there is English translation in the Vietnamese language translation file :)

Reply:
Vietnamese use english rather regulary in their website 👍